eudemo/eudemo/reactor.py

```diff
@@ -127,6 +131,49 @@ def build_tf_coils(
     return TFCoil(builder.build(), builder._make_field_solver())
 
 
+def build_pf_coils(params, build_config, tf_coil_boundary, pf_coil_keep_out_zones=()):
+    """
+    Design and build the PF coils for the reactor.
+    """
+    pf_designer = PFCoilsDesigner(
+        params,
+        build_config,
+        tf_coil_boundary,
+        pf_coil_keep_out_zones,
+    )
+    coilset = pf_designer.execute()
+
+    wires = []
+    for name in coilset.name:
+        if not (coilset[name].dx == 0 or coilset[name].dz == 0):
+            wires.append(
+                (PFCoilPictureFrame(params, coilset[name]), coilset[name].ctype)
+            )
+
+    builders = []
+    for (des, ctype) in wires:
```

Comment:
Another very minor thing, but I feel we can afford to be more explicit here
```suggestion
    for (designer, coil_type) in wires:
```